bin/cli.js

```diff
@@ -48,7 +47,6 @@ const cliFlags = yargs
   })
   .option('config', {
     'describe': 'Path to json config file',
-    'coerce': (arg) => getConfigFromFile(arg),
```

Comment:
I was merely trying to take advantage of the ```yargs``` library as much as possible, not particularly critical IMO.